docs/configuration-files.md

```diff
@@ -161,7 +161,7 @@ See the [`patterns`](configuration-files.md#patterns) example above.
 
 ### `pattern-inside`
 
-The `pattern-inside` operator keeps matched findings that reside within its expression. This is useful for finding code inside other pieces of code like functions or if blocks.
+The `pattern-inside` operator keeps matched findings that reside within its expression. This is useful for finding code inside other pieces of code like functions or if blocks. At the moment, your rule can have at most pattern-inside clause.
```

Comment:
@ulziibay I'm not sure I understand ` At the moment, your rule can have at most pattern-inside clause`? Did you mean at most 1 pattern inside or something about 1 level of nested clauses?

Reply:
Clarified to 
`At the moment, your rule can have at most one pattern-inside clause`